.github/workflows/clp-core-build.yaml

```
@@ -247,6 +280,45 @@
+      matrix:
+        use_shared_libs: [true, false]
+    name: "ubuntu-noble-${{matrix.use_shared_libs == true && 'dynamic' || 'static'}}-linked-bins"
+    continue-on-error: true
+    runs-on: "ubuntu-latest"
+    steps:
+      - uses: "actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683"
+        with:
+          submodules: "recursive"
+
+      - name: "Work around actions/runner-images/issues/6775"
+        run: "chown $(id -u):$(id -g) -R ."
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2046:warning:1:7: Quote this to prevent word splitting

(shellcheck)